source/ui/ui-about.cpp

```diff
@@ -62,15 +62,16 @@ static const std::vector<std::string_view> _thankyous = {
 
 streamfx::ui::about::about() : QDialog(reinterpret_cast<QWidget*>(obs_frontend_get_main_window()))
 {
-	std::deque<ui::about::entry> entries;
+	std::vector<ui::about::entry> entries;
```

Comment:
`std::vector` is suboptimal if the maximum size is not known ahead of time. Either initialize the vector, or revert the change.